spark/v4.0/spark/src/main/scala/org/apache/spark/sql/stats/ThetaSketchAgg.scala

```diff
@@ -119,3 +122,12 @@ case class ThetaSketchAgg(
     compactSketch.toByteArray
   }
 }
+
+object ThetaSketchAgg {
+  def expr(node: ColumnNode): Expression = {
+    node match {
+      case ExpressionColumnNode(expression, _) => expression
+      case node => throw SparkException.internalError("Unsupported ColumnNode: " + node)
```

Comment:
Should we be throwing a Spark Internal error here? Seems like this our issue?

Reply:
I will change this to an Iceberg Exception. I am not making the change in this round because I want to try Preview 1 to see if the other changes can pass the CI. I will fix this later when I try Preview 2.